ibis/tests/all/test_window.py

```diff
@@ -323,25 +329,120 @@ def test_bounded_preceding_windows(backend, alltypes, df, con, window_fn):
         ),
     ],
 )
+@pytest.mark.parametrize(
+    ('ordered'), [param(True, id='orderered'), param(False, id='unordered')],
```

Comment:
are these ids different for a reason? why are you using param here?

Reply:
I think this actually doesn't need `param` or ids. I'll change it to just `[True, False]`

Reply:
Actually this makes the test output a bit nicer, are you OK with keeping this as is?
```
# Without ids
XFAIL ibis/tests/all/test_window.py::test_unbounded_window_grouped[Pandas-True-mean]
```
vs
```
# With ids
XFAIL ibis/tests/all/test_window.py::test_unbounded_window_grouped[Pandas-orderered-mean]
```